tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -60,6 +62,69 @@ def _ValidateType(self):
 
     return True
 
+  def _ValidateTranslation(self):
+    """Uses information from the generated ontology universe to validate
+    an entity's translation if it exists.
+
+    Returns:
+      Returns boolean for validity of entity translation, defaults to True if
```

Comment:
I think here we need to default to false?

Reply:
The `translation` key is optional and not necessary for a building config to be valid. So the default if it doesn't exist is `True`, and we need to validate it only if it exists.